src/app/data/consts.js

```diff
@@ -0,0 +1,33 @@
+class Consts {
+  static GAME_MODES = {
+    students: {
+      gamemode: 'students',
```

Comment:
"Students" with big first letter will be easier to read from this and inject prepared text into website

Reply:
Remember That you can also manipulate letter casing within your code. In general, keys usually start with lowercase, but that is not a must.